packages/gatsby-dev-cli/src/watch.js

```diff
@@ -17,7 +17,7 @@ function watch(root, packages, { scanOnce, quiet }) {
     const ignoreRegs = [
       /[\/\\]node_modules[\/\\]/i,
       /\.git/i,
-      new RegExp(`${p}[\\/]src[\\/]`, `i`),
+      new RegExp(`${p}[\\/\\\\]src[\\/\\\\]`, `i`),
```

Comment:
@jquense I think I get it. But you meant to write `\\windows\paths\use\these`, right?